dcc-portal-ui/app/scripts/common/js/location.js

```diff
@@ -138,6 +138,26 @@
         delete s[param];
         $location.search(s);
       },
+      getPaginationParam: function(dataType){
+        var params = {},
```

Comment:
Let's also keep the plurality consistent
`param` vs `params`
Seems like the function is returning "params", let's call it `getPaginationParams`